episodes/03-create.md

```diff
@@ -17,12 +17,40 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## The `Git` jargon
+
+`Git` is a topic that contains a lot of words to do version control.
+
+![Word cloud for Git from <https://thoughtbot.com/blog/recommending-blog-posts>](fig/topic-git.png)
+
+We will locate them using this workflow bellow as template. We will relate Version control _actions_ that we can perform with specific _`git verb` commands_. These verbs will record your changes between `Git` _spaces_ associated to your folder.
+
+![Workflow will show actions, `git verb` commands, and spaces.](fig/cut-git-verb_map-00.png)
+
+In this episode, we are going to learn how to initialize Git to create a Local Repository in our folder, also known as Working directory or Workspace.
+
+![Initialize a Local Repository in your Workspace with the `git init` command verb](fig/cut-git-verb_map-01.png)
+
+Let's start with a new R project in Rstudio.
```

Comment:
[RStudio](https://posit.co/products/open-source/rstudio/) has a capital S, change all instances of lower case s.

Reply:
sutil detail! I cited this as issue in #23